test/performance/shmem_perf_suite/bi_dir.h

```diff
@@ -78,7 +78,7 @@ static inline void bi_bw_put(int len, perf_metrics_t *metric_info)
         shmem_int_p(&fin, 1, dest);
         shmem_int_wait_until(&fin, SHMEM_CMP_EQ, 0);
         end = perf_shmemx_wtime();
-        calc_and_print_results(end, start, len, *metric_info);
+        calc_and_print_results(end, start, len, 1, *metric_info);
```

Comment:
personally I'd prefer bundling this into metric_info calling it num_partners and setting one as default (override it for target), it would be easier to read

Reply:
Sure, will do.